packages/js/product-editor/src/blocks/product-fields/product-details-section-description/editor.scss

```diff
@@ -0,0 +1,40 @@
+.wp-block-woocommerce-product-details-section-description {
+	margin-top: $grid-unit;
+	display: flex;
+	justify-content: flex-start;
+	align-items: center;
+	flex-wrap: wrap;
+	gap: $grid-unit-05;
+
+	p {
+		margin: 0;
+	}
+
+	&__dropdown .components-button.components-menu-item__button {
+		border-radius: 2px;
+		padding: $grid-unit $grid-unit + 2;
```

Comment:
Why the `+2`? Seems odd to be hardcoding a value and using variables as well.

Reply:
Well the unit is not part of the one defined by the grid system. But I prefer to use relative ones than hard coded ones. But if you think we should just set the hard coded number there I can do it too.

Reply:
It just seems like using a combination of grid system variables and relative values is not robust. Unless we really always want the padding to be `$grid-unit + 2` regardless of what `$grid-unit` might be in the future.

I think ideally if we are going to use the grid system we should use it exclusively.

I'm fine with keeping this as is though.